frontend/packages/dev-console/integration-tests/support/commands/app.ts

```diff
@@ -25,7 +28,26 @@ after(() => {
 });
 
 afterEach(() => {
-  // checkErrors();
+  checkErrors();
```

Comment:
It's commented purposefully due to issue occurred in other pr's, which needs to be identified. Before removing comment.
Reference: https://github.com/openshift/console/pull/7751

Reply:
sure.